### PR TITLE
MAUI-5256 : Prepare Localization UG for Popup Control

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared Localization UG for Popup Control

### CODE REVIEW

File: MAUI/Popup/localization.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+layout: post
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:2]: Layout: Post

File: MAUI/Popup/localization.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+title: Localization in .NET MAUI SfPopup control | Syncfusion
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:3]: Title: Localization in .NET MAUI SfPopup control | Syncfusion

File: MAUI/Popup/localization.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+description: Learn here all about Localization support in Syncfusion .NET MAUI Popup (SfPopup) control and more.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:4]: Description: Learn about Localization support in Syncfusion .NET MAUI Popup (SfPopup) control and more.

File: MAUI/Popup/localization.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+platform: MAUI
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:5]: Platform: MAUI

File: MAUI/Popup/localization.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+control: SfPopup
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:6]: Control: SfPopup

File: MAUI/Popup/localization.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+documentation: ug
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:7]: Documentation: UG

File: MAUI/Popup/localization.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+# Localization in .NET MAUI Popup (SfPopup)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfPopup` can be localized by adding `resource` file.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:12]: Localization is the process of translating the application resources into different languages for specific cultures. The `SfPopup` can be localized by adding a `resource` file.

File: MAUI/Popup/localization.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+documentation: ug
+---
+
+# Localization in .NET MAUI Popup (SfPopup)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfPopup` can be localized by adding `resource` file. 
+
+## Setting CurrentUICulture to the application
+
+Application culture can be changed by setting `CurrentUICulture.` in `App.xaml.cs` file.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:16]: Application culture can be changed by setting the `CurrentUICulture` in the `App.xaml.cs` file.

File: MAUI/Popup/localization.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The required `resx` files with `Build Action` as `EmbeddedResource` (File name should contain culture code) into the `Resources` folder.
+
+## Localize application level
+
+To localize the `Popup` based on `CurrentUICulture` using `resource` files, follow the below steps.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:46]: To localize the `Popup` based on the `CurrentUICulture` using the `resource` files, follow these steps.

File: MAUI/Popup/localization.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+   1. Create new folder, named as `Resources` in the application.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:48]: 1. Create a new folder named `Resources` in the application.

File: MAUI/Popup/localization.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+   2. Right-click on the `Resources` folder, select `Add` and then `NewItem.`
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:50]: 2. Right-click on the `Resources` folder, select `Add,` then `NewItem.`

File: MAUI/Popup/localization.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+   3. In Add New Item wizard, select the Resource File option and name the filename as `SfPopup.<culture name>.resx.` For example, give the name as `SfPopup.fr-FR.resx` for French culture.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:52]: 3. In the Add New Item wizard, select the Resource File option and name the filename as `SfPopup.<culture name>.resx.` For example, give the name as `SfPopup.fr-FR.resx` for French culture.

File: MAUI/Popup/localization.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+   4. The culture name indicates the name of the language and country.
+
+   ![shows-the-name-of-resource-file-to-be-added-for-maui-popup](/Images/localization/shows-the-name-of-resource-file-to-be-added-for-maui-popup.png)
+
+   5. Now, select `Add` option to add the resource file in **Resources** folder.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:58]: 5. Now, select `Add` option to add the resource file in the **Resources** folder.

File: MAUI/Popup/localization.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+   ![shows-the-added-resource-file-for-french-language-in-maui-popup](/Images/localization/shows-the-added-resource-file-for-french-language-in-maui-popup.png)
+
+   6. Add the Name/Value pair in Resource Designer of `SfPopup.fr-FR.resx` file and change its corresponding value to corresponding culture.
```
Comment (Chrispineagunja) [inline MAUI/Popup/localization.md:62]: 6. Add the Name or Value pair in the Resource Designer of `SfPopup.fr-FR.resx` file and change its corresponding value to corresponding culture.

File: MAUI/Popup/localization.md (hunk 13 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:58]: modified

File: MAUI/Popup/localization.md (hunk 12 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:52]: modified

File: MAUI/Popup/localization.md (hunk 10 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:48]: modified

File: MAUI/Popup/localization.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:5]: not changed

File: MAUI/Popup/localization.md (hunk 3 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:4]: modified

File: MAUI/Popup/localization.md (hunk 2 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:3]: not changed

File: MAUI/Popup/localization.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/Popup/localization.md:2]: not changed

**Pull request closed without merging**